main/php_ini.c

```diff
@@ -348,6 +354,31 @@ static void php_load_php_extension_cb(void *arg)
 }
 /* }}} */
 
+static void *attempt_dl_load(char *path, char **errp)
```

Comment:
I wanted to avoid calling ext/standard from 'main'. As I didn't know if those parts could communicate with eaach other, I did two identical static functions. If a function in ext/standard can be loaded from main, I'll do it.